spec/license_meta_spec.rb

```diff
@@ -43,9 +45,38 @@
           'zlib'
         ]
         skip 'added before 3 using examples required' if legacy.include?(license['slug'])
-        examples = raw_fields['using'] || []
         expect(examples.length).to eq(3)
       end
+
+      context 'licensee detects using examples' do
+        module Licensee
+          class License
+            class << self
+              def license_dir
+                dir = ::File.dirname(__FILE__)
+                ::File.expand_path '../_licenses', dir
+              end
+            end
+          end
+        end
+
+        examples.each do |example|
+          example_url = example.values[0]
+          if example_url.index('https://github.com/') == 0
```

Comment:
You may also be able to use `include?` or `start_with?` if you think it's easier to read.